@vorburger I tried to understand the error, but either there is not enough information on what caused it, or I am missing something obvious? 

I am pasting the part of the log here, can you guide me in it? - It is related to the issue: https://issues.apache.org/jira/browse/FINERACT-927


org.apache.fineract.integrationtests.SchedulerJobsTest > testTriggeringManualExecutionOfAllSchedulerJobs FAILED
    java.lang.AssertionError: 1 expectation failed.
    Expected status code <202> but was <500>.
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
        at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)
        at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:59)
        at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
        at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:493)
        at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
        at io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1656)
        at jdk.internal.reflect.GeneratedMethodAccessor84.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
        at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1262)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1029)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:816)
        at groovy.lang.GroovyObject.invokeMethod(GroovyObject.java:39)
        at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:45)
        at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:55)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:203)
        at io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1661)
        at jdk.internal.reflect.GeneratedMethodAccessor83.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
        at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1262)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1029)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:816)
        at groovy.lang.GroovyObject.invokeMethod(GroovyObject.java:39)
        at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:45)
        at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:55)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:203)
        at io.restassured.internal.RequestSpecificationImpl.post(RequestSpecificationImpl.groovy:175)
        at io.restassured.internal.RequestSpecificationImpl.post(RequestSpecificationImpl.groovy)
        at org.apache.fineract.integrationtests.common.Utils.performServerPost(Utils.java:165)
        at org.apache.fineract.integrationtests.common.SchedulerJobHelper.updateSchedulerStatus(SchedulerJobHelper.java:105)
        at org.apache.fineract.integrationtests.SchedulerJobsTest.tearDown(SchedulerJobsTest.java:62)
org.apache.fineract.integrationtests.SchedulerJobsTest > testFlippingSchedulerStatus SKIPPED

--- a/fineract-provider/config/checkstyle/checkstyle.xml
+++ b/fineract-provider/config/checkstyle/checkstyle.xml
@@ -120,7 +120,7 @@
             <message key="forbid.certain.imports" value="Use ''java.nio.charset.StandardCharsets'' instead of ''{0}''" />
         </module>
       
-        <module name="SuppressWarningsHolder" />
+
         <module name="OuterTypeFilename"/>
         <module name="OneTopLevelClass"/>
        

